hazelcast/include/hazelcast/client/serialization/pimpl/compact.h

```diff
@@ -1407,6 +1407,13 @@ class HAZELCAST_API default_compact_writer
 
 struct HAZELCAST_API field_descriptor
 {
+    field_descriptor(
```

Comment:
implementation in header? why?

Reply:
Parameters are just defaulted. It is not implementation.